rest-service/manager_rest/security/user_handler.py

```diff
@@ -43,6 +44,10 @@ def user_loader(request):
     """
     if request.authorization:
         return get_user_from_auth(request.authorization)
+    execution_token = get_execution_token_from_request(request)
+    if execution_token:
+        execution = get_current_execution_by_token(execution_token)
```

Comment:
this has to be here, can't be inside of the authentication.py instead? currently the auth with execution token flow is split between those two places, harder to follow

Reply:
Yes I agree, but the current_user assignment is here